treeherder/perf/auto_perf_sheriffing/telemetry_alerting/email_manager.py

```diff
@@ -0,0 +1,134 @@
+        content.write_email(probe, alert)
+        self._email.content = str(content)
+
+
+class TelemetryEmailContent:
+    DESCRIPTION = (
+        "MozDetect has detected a telemetry change in a probe you are subscribed to:\n---\n"
+    )
+
+    TABLE_HEADERS = (
+        "| Channel | Probe | Platform | Date Range | Detection Push |\n"
+        "| :---: | :---: | :---: | :---: | :---: |\n"
+    )
+
+    ADDITIONAL_PROBES = (
+        "See below for additional probes that alerted at the same time (or near "
+        "the same push):"
+        "\n---\n"
+    )
+
+    def __init__(self):
+        self._raw_content = None
+
+    def write_email(self, probe, alert):
+        self._initialize_report_intro()
+        self._include_probe(
+            alert.get_detection_range(), alert.telemetry_signature, alert.telemetry_alert_summary
+        )
+
+        if alert.get_related_alerts():
+            self._include_additional_probes(alert)
+
+    def _initialize_report_intro(self):
+        if self._raw_content is None:
+            self._raw_content = self.DESCRIPTION + self.TABLE_HEADERS
+
+    def _include_probe(self, detection_range, telemetry_signature, telemetry_alert_summary):
+        new_table_row = self._build_table_row(
+            detection_range, telemetry_signature, telemetry_alert_summary
+        )
+        self._raw_content += f"{new_table_row}\n"
+
+    def _include_additional_probes(self, alert):
+        self._raw_content += f"\n{self.ADDITIONAL_PROBES}{self.TABLE_HEADERS}"
+        for related_alert in alert.get_related_alerts():
+            self._include_probe(
+                alert.get_detection_range(),
+                related_alert.series_signature,
+                alert.telemetry_alert_summary,
+            )
+
+    def _build_table_row(
+        self, detection_range, telemetry_signature, telemetry_alert_summary
+    ) -> str:
+        # TODO: Have change-detection-technique/mozdetect provide a method for building
+        # a row. That way we can decouple the information provided to bugzilla
+        # users from the alerting system.
+        return (
+            "| {channel} | [{probe}]({glean_dictionary_link}) | {platform} "
+            "| [{date_from} - {date_to}]({treeherder_date_link})"
+            "| [Detection Push]({treeherder_push_link}) |"
+        ).format(
+            channel=telemetry_signature.channel,
+            probe=telemetry_signature.probe,
+            platform=telemetry_signature.platform,
+            date_from=detection_range["from"].time.strftime("%Y-%m-%d"),
+            date_to=detection_range["to"].time.strftime("%Y-%m-%d"),
+            treeherder_date_link=get_treeherder_detection_range_link(
+                detection_range, telemetry_signature
+            ),
+            treeherder_push_link=get_treeherder_detection_link(
+                detection_range, telemetry_signature
+            ),
+            glean_dictionary_link=get_glean_dictionary_link(telemetry_signature),
```

Comment:
NIT: to keep the list in order and have the glean_dictionary_link be after the probe and before the platform value